src/WorkItemMigrator/WorkItemImport/WitClient/IWitClientWrapper.cs

```diff
@@ -8,7 +9,7 @@ namespace WorkItemImport
 {
     public interface IWitClientWrapper
     {
-        WorkItem CreateWorkItem(string wiType);
+        WorkItem CreateWorkItem(string wiType, DateTime createdDate = default, string createdBy = "");
```

Comment:
use null instead of default

Reply:
Fixed by https://github.com/solidify/jira-azuredevops-migrator/pull/648/commits/4f6ee65a7c91eaa61a952ca35f3b40df31fcb569